app/models/manual_publish_task.rb

```diff
@@ -1,4 +1,4 @@
-require "state_machine"
+require "state_machines"
```

Comment:
Do we definitely need this `require` line?

The `SectionEdition` model seems to work okay without the explicit require.